controller_manager:
  ros__parameters:
    update_rate: 100
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster
    joint_trajectory_controller:
      type: joint_trajectory_controller/JointTrajectoryController
    left_wheel_controller:
      type: forward_command_controller/ForwardCommandController
    right_wheel_controller:
      type: forward_command_controller/ForwardCommandController

joint_trajectory_controller:
  ros__parameters:
    joints:
      - shoulder_joint
      - elbow_joint

    command_interfaces:
      - position

    state_interfaces:
      - position
    
    state_publish_rate: 50.0
    action_monitor_rate: 20.0

    allow_partial_joints_goal: false
    open_loop_control: true
    constraints:
      stopped_velocity_tolerance: 0.01
      goal_time: 0.0
      joint1:
        trajectory: 0.05
        goal: 0.03

left_wheel_controller:
  ros__parameters:
    joints:
      - left_wheel_joint
    interface_name: velocity

right_wheel_controller:
  ros__parameters:
    joints:
      - right_wheel_joint
    interface_name: velocity
